go/workspace.rst

```diff
@@ -0,0 +1,208 @@
+
+Go workspace rules
+==================
+
+.. _github.com/google/protobuf: https://github.com/google/protobuf/
+.. _github.com/golang/protobuf: https://github.com/golang/protobuf/
+.. _google.golang.org/grpc: https://github.com/grpc/grpc-go
+.. _golang.org/x/net: https://github.com/golang/net/
+.. _golang.org/x/tools: https://github.com/golang/tools/
+.. _go_library: core.rst#go_library
+.. _toolchains: toolchains.rst
+.. _go_register_toolchains: toolchains.rst#go_register_toolchains
+.. _go_sdk: toolchains.rst#go_sdk
+.. _go_toolchain: toolchains.rst#go_toolchain
+.. _new_go_repository: deprecated.rst#new_go_repository
+.. _normal go logic: https://golang.org/cmd/go/#hdr-Remote_import_paths
+.. _gazelle: tools/gazelle/README.md
+.. _http_archive: https://docs.bazel.build/versions/master/be/workspace.html#http_archive
+.. _git_repository: https://docs.bazel.build/versions/master/be/workspace.html#git_repository
+.. _nested workspaces: https://bazel.build/designs/2016/09/19/recursive-ws-parsing.html
+
+.. _go_prefix_faq: /README.rst#whats-up-with-the-go_default_library-name
+.. |go_prefix_faq| replace:: FAQ
+
+.. role:: param(kbd)
+.. role:: type(emphasis)
+.. role:: value(code)
+.. |mandatory| replace:: **mandatory value**
+
+Workspace rules are either repository rules, or macros that are intended to be used from the
+WORKSPACE file.
+
+`Toolchain support <toolchains>`_:
```

Comment:
Since these are documented on another page, consider changing this paragraph to be a "see also" at the bottom of this section.